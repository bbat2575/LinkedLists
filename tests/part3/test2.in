NEW 1
0
NEW 1
1
NEW 1
2
NEW 1
3
NEW 1
4
NEW 1
5
NEW 1
6
NEW 1
7
NEW 1
8
NEW 1
9
NEW 1
10
NEW 1
11
NEW 1
12
NEW 1
13
NEW 1
14
NEW 1
15
NEW 1
16
NEW 1
17
NEW 1
18
NEW 1
19
NEW 1
20
NEW 1
21
NEW 1
22
NEW 1
23
NEW 1
24
NEW 1
25
NEW 1
26
NEW 1
27
NEW 1
28
NEW 1
29
NEW 1
30
NEW 1
31
NEW 1
32
NEW 1
33
NEW 1
34
NEW 1
35
NEW 1
36
NEW 1
37
NEW 1
38
NEW 1
39
NEW 1
40
NEW 1
41
NEW 1
42
NEW 1
43
NEW 1
44
NEW 1
45
NEW 1
46
NEW 1
47
NEW 1
48
NEW 1
49
NEW 1
50
NEW 1
51
NEW 1
52
NEW 1
53
NEW 1
54
NEW 1
55
NEW 1
56
NEW 1
57
NEW 1
58
NEW 1
59
NEW 1
60
NEW 1
61
NEW 1
62
NEW 1
63
NEW 1
64
NEW 1
65
NEW 1
66
NEW 1
67
NEW 1
68
NEW 1
69
NEW 1
70
NEW 1
71
NEW 1
72
NEW 1
73
NEW 1
74
NEW 1
75
NEW 1
76
NEW 1
77
NEW 1
78
NEW 1
79
NEW 1
80
NEW 1
81
NEW 1
82
NEW 1
83
NEW 1
84
NEW 1
85
NEW 1
86
NEW 1
87
NEW 1
88
NEW 1
89
NEW 1
90
NEW 1
91
NEW 1
92
NEW 1
93
NEW 1
94
NEW 1
95
NEW 1
96
NEW 1
97
NEW 1
98
NEW 1
99
NEW 100
{0}
{1}
{2}
{3}
{4}
{5}
{6}
{7}
{8}
{9}
{10}
{11}
{12}
{13}
{14}
{15}
{16}
{17}
{18}
{19}
{20}
{21}
{22}
{23}
{24}
{25}
{26}
{27}
{28}
{29}
{30}
{31}
{32}
{33}
{34}
{35}
{36}
{37}
{38}
{39}
{40}
{41}
{42}
{43}
{44}
{45}
{46}
{47}
{48}
{49}
{50}
{51}
{52}
{53}
{54}
{55}
{56}
{57}
{58}
{59}
{60}
{61}
{62}
{63}
{64}
{65}
{66}
{67}
{68}
{69}
{70}
{71}
{72}
{73}
{74}
{75}
{76}
{77}
{78}
{79}
{80}
{81}
{82}
{83}
{84}
{85}
{86}
{87}
{88}
{89}
{90}
{91}
{92}
{93}
{94}
{95}
{96}
{97}
{98}
{99}
VIEW-NESTED 100
TYPE 100
