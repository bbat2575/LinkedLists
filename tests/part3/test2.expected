
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 0

> List 1: 1

> List 2: 2

> List 3: 3

> List 4: 4

> List 5: 5

> List 6: 6

> List 7: 7

> List 8: 8

> List 9: 9

> List 10: 10

> List 11: 11

> List 12: 12

> List 13: 13

> List 14: 14

> List 15: 15

> List 16: 16

> List 17: 17

> List 18: 18

> List 19: 19

> List 20: 20

> List 21: 21

> List 22: 22

> List 23: 23

> List 24: 24

> List 25: 25

> List 26: 26

> List 27: 27

> List 28: 28

> List 29: 29

> List 30: 30

> List 31: 31

> List 32: 32

> List 33: 33

> List 34: 34

> List 35: 35

> List 36: 36

> List 37: 37

> List 38: 38

> List 39: 39

> List 40: 40

> List 41: 41

> List 42: 42

> List 43: 43

> List 44: 44

> List 45: 45

> List 46: 46

> List 47: 47

> List 48: 48

> List 49: 49

> List 50: 50

> List 51: 51

> List 52: 52

> List 53: 53

> List 54: 54

> List 55: 55

> List 56: 56

> List 57: 57

> List 58: 58

> List 59: 59

> List 60: 60

> List 61: 61

> List 62: 62

> List 63: 63

> List 64: 64

> List 65: 65

> List 66: 66

> List 67: 67

> List 68: 68

> List 69: 69

> List 70: 70

> List 71: 71

> List 72: 72

> List 73: 73

> List 74: 74

> List 75: 75

> List 76: 76

> List 77: 77

> List 78: 78

> List 79: 79

> List 80: 80

> List 81: 81

> List 82: 82

> List 83: 83

> List 84: 84

> List 85: 85

> List 86: 86

> List 87: 87

> List 88: 88

> List 89: 89

> List 90: 90

> List 91: 91

> List 92: 92

> List 93: 93

> List 94: 94

> List 95: 95

> List 96: 96

> List 97: 97

> List 98: 98

> List 99: 99

> Nested 100: {List 0} -> {List 1} -> {List 2} -> {List 3} -> {List 4} -> {List 5} -> {List 6} -> {List 7} -> {List 8} -> {List 9} -> {List 10} -> {List 11} -> {List 12} -> {List 13} -> {List 14} -> {List 15} -> {List 16} -> {List 17} -> {List 18} -> {List 19} -> {List 20} -> {List 21} -> {List 22} -> {List 23} -> {List 24} -> {List 25} -> {List 26} -> {List 27} -> {List 28} -> {List 29} -> {List 30} -> {List 31} -> {List 32} -> {List 33} -> {List 34} -> {List 35} -> {List 36} -> {List 37} -> {List 38} -> {List 39} -> {List 40} -> {List 41} -> {List 42} -> {List 43} -> {List 44} -> {List 45} -> {List 46} -> {List 47} -> {List 48} -> {List 49} -> {List 50} -> {List 51} -> {List 52} -> {List 53} -> {List 54} -> {List 55} -> {List 56} -> {List 57} -> {List 58} -> {List 59} -> {List 60} -> {List 61} -> {List 62} -> {List 63} -> {List 64} -> {List 65} -> {List 66} -> {List 67} -> {List 68} -> {List 69} -> {List 70} -> {List 71} -> {List 72} -> {List 73} -> {List 74} -> {List 75} -> {List 76} -> {List 77} -> {List 78} -> {List 79} -> {List 80} -> {List 81} -> {List 82} -> {List 83} -> {List 84} -> {List 85} -> {List 86} -> {List 87} -> {List 88} -> {List 89} -> {List 90} -> {List 91} -> {List 92} -> {List 93} -> {List 94} -> {List 95} -> {List 96} -> {List 97} -> {List 98} -> {List 99}

> {0} -> {1} -> {2} -> {3} -> {4} -> {5} -> {6} -> {7} -> {8} -> {9} -> {10} -> {11} -> {12} -> {13} -> {14} -> {15} -> {16} -> {17} -> {18} -> {19} -> {20} -> {21} -> {22} -> {23} -> {24} -> {25} -> {26} -> {27} -> {28} -> {29} -> {30} -> {31} -> {32} -> {33} -> {34} -> {35} -> {36} -> {37} -> {38} -> {39} -> {40} -> {41} -> {42} -> {43} -> {44} -> {45} -> {46} -> {47} -> {48} -> {49} -> {50} -> {51} -> {52} -> {53} -> {54} -> {55} -> {56} -> {57} -> {58} -> {59} -> {60} -> {61} -> {62} -> {63} -> {64} -> {65} -> {66} -> {67} -> {68} -> {69} -> {70} -> {71} -> {72} -> {73} -> {74} -> {75} -> {76} -> {77} -> {78} -> {79} -> {80} -> {81} -> {82} -> {83} -> {84} -> {85} -> {86} -> {87} -> {88} -> {89} -> {90} -> {91} -> {92} -> {93} -> {94} -> {95} -> {96} -> {97} -> {98} -> {99}

> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference -> reference

> 
